localization/beluga_demo_ndt_localization/launch/demo_hallway_ndt_localization.launch.py

```diff
@@ -0,0 +1,136 @@
+            ),
+            DeclareLaunchArgument(
+                name="world_name",
+                default_value="magazino_hallway.world",
+                description="Name of the world file to load in simulation",
+                choices=available_worlds,
+            ),
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    PathJoinSubstitution(
+                        [
+                            FindPackageShare("beluga_example"),
+                            "launch",
+                            "utils",
+                            "ndt_3d_localization_launch.py",
+                        ]
+                    ),
+                ),
+                launch_arguments={
+                    'localization_ndt_map': LaunchConfiguration('localization_ndt_map')
+                }.items(),
+            ),
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    PathJoinSubstitution(
+                        [
+                            FindPackageShare("beluga_example"),
+                            "launch",
+                            "utils",
+                            "rviz_launch.py",
+                        ]
+                    ),
+                ),
+                launch_arguments={
+                    'user_sim_time': 'true',
+                    # 'display_config': LaunchConfiguration('localization_params_file'),
+                }.items(),
+            ),
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    [
+                        os.path.join(
+                            get_package_share_directory("gazebo_ros"),
+                            "launch",
+                            "gazebo.launch.py",
+                        )
+                    ],
+                ),
+                launch_arguments=[("world", world_path)],
+            ),
+            Node(
+                package="gazebo_ros",
+                executable="spawn_entity.py",
+                arguments=[
+                    "-topic",
+                    "robot_description",
+                    "-entity",
+                    "gonbuki_robot",
+                    "-x",
+                    "0.0",
+                    "-y",
+                    "2.0",
+                ],
+                output="screen",
+            ),
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    PathJoinSubstitution(
+                        [
+                            FindPackageShare("gonbuki_description"),
+                            "launch",
+                            "robot_description.launch.py",
+                        ]
+                    ),
+                ),
+                launch_arguments={'world_name': 'magazino_hallway.world'}.items(),
+            ),
+            Node(
```

Comment:
maybe you can use https://github.com/ekumenlabs/kobuki_ros2_stack/blob/main/bringup/gonbuki_bringup_common/launch/bringup_launch.py and don't include many of the nodes here (teleop, rviz, simulation, spawn)